README.md

```diff
@@ -22,6 +22,7 @@ At Babylon, we firmly believe that **transparency** is a core value that should
 | Adam Borek              | [@TheAdamBorek](https://twitter.com/TheAdamBorek)             |
 | Ana Catarina Figueiredo | [@AnnKatFig](https://twitter.com/AnnKatFig)                   |
 | Anders Ha               | [@_andersha](https://twitter.com/_andersha)                   |
+| Anil Puttabuddhi        |                                                               |
```

Comment:
@anilputtabuddhi  I have no issue if you don't have a Twitter or don't want to share it, but would you be open to a link to your GitHub profile instead, as some other people did? (it helps when we need to mention people in PRs and don't know their GH handle) – It's no obligation, I'm just suggesting 😉

Reply:
@AliSoftware I do have a twitter account on which I tweeted in 2010 I guess :).  I didn't realise I could put Github handle on it on a cursory glance. Sure will add my Github handle in there.